src/lib/Scenes/Show/Components/ShowHeader/Components/ArtistNamesList.tsx

```diff
@@ -1,33 +1,52 @@
-import { Sans } from "@artsy/palette"
+import { Flex, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import React from "react"
+import { TouchableOpacity } from "react-native"
 
-export const ArtistNamesList = ({ artists }) => {
+export const ArtistNamesList = ({ artists, Component, viewAllArtists }) => {
   if (!artists || !artists.length) {
     return null
   }
+  const artistCutoffIndex = 5
+  const artistsToDisplay = []
+  const truncatedArtistList = artists.slice(0, artistCutoffIndex)
+  truncatedArtistList.map((artist, index) => {
+    artistsToDisplay.push(
+      <TouchableOpacity
+        key={index}
+        onPress={() => {
+          SwitchBoard.presentNavigationViewController(Component, artist.href)
+        }}
+      >
+        <Sans size="2" weight="medium">
+          {index === truncatedArtistList.length - 1 ? artist.name : artist.name + ", "}
+        </Sans>
+      </TouchableOpacity>
+    )
+  })
 
-  const names = artists
-    .map(artist => artist.name)
-    .slice(0, 5)
-    .join(", ")
   const remainderCount = artists.length - 5
-  const remainder =
+  const remainingArtists =
     remainderCount > 0 ? (
-      <>
-        {" and "}
-        <Sans size="2" weight="medium">
-          {remainderCount} more
-        </Sans>
-      </>
+      <Flex flexDirection="row" flexWrap="wrap">
+        <Sans size="2"> and </Sans>
+        <TouchableOpacity
+          onPress={() => {
+            viewAllArtists()
+          }}
+        >
+          <Sans size="2" weight="medium">
+            {remainderCount + " more"}
+          </Sans>
+        </TouchableOpacity>
+      </Flex>
     ) : null
```

Comment:
No neeed for `null` / ternary, since JSX won't render falsy values; can use a guard instead:

```jsx
const remainingArtists = remainderCount > 0 && (
  <Flex>... 
)